config/manifests/bases/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -32,26 +32,16 @@ spec:
         kind: RateLimitPolicy
         name: ratelimitpolicies.kuadrant.io
         version: v1beta3
-      - description: DNSHealthCheckProbe enables performing health checks against a DNS endpoint (A or CNAME record)
-        displayName: DNSHealthCheckProbe
-        kind: DNSHealthCheckProbe
-        name: dnshealthcheckprobes.kuadrant.io
-        version: v1alpha1
       - description: DNSPolicy configures how North-South based traffic should be balanced and reach the gateways
         displayName: DNSPolicy
         kind: DNSPolicy
         name: dnspolicies.kuadrant.io
-        version: v1alpha1
-      - description: DNSRecord represents DNS endpoints for gateway network services
-        displayName: DNSRecord
-        kind: DNSRecord
```

Comment:
Does this mean the `DNSRecord` and `DNSHealthCheckProbe` CRs will be removed? Or it is defined elsewhere?